Makefile

```diff
@@ -68,6 +68,7 @@ install:: provider
 		yarn install --offline --production && \
 		(yarn unlink > /dev/null 2>&1 || true) && \
 		yarn link
+	cd ${PACKDIR}/python/bin && $(PIP) install --user -e .
```

Comment:
I don't think we do the equivalent of this in other repos - was there a reason you wanted to add it here?  If so, should we be consistent in other repos?

Reply:
This happens in *some* repos

Reply:
This is from the original #160 PR, I didn't add it. I don't have a strong opinion here, except that I agree that we should be consistent across repos.